network/network.go

```diff
@@ -53,6 +53,25 @@ const (
 	CannotConnect
 )
 
+// Reachability indicates how reachable a node is.
+type Reachability int
```

Comment:
Just for clarification is the reason we're put the enum in the network package instead of the event package because we're hoping reachability will be more general than just the `EvtLocalReachabilityChanged` event?

Reply:
Yes. Basically, it's not specific to the event itself.